### PR TITLE
chore(scripts): make quick installer script POSIX sh compliant

### PR DESCRIPTION
**Pull request by profclems**
Currently the quick install script isn't POSIX sh compliant and hence, works only
on bash.

In order to make it run on all shell types, it must be POSIX sh compliant.
All [bashisms][0] have been checks and removed/fixed to POSIX sh formats.

[0]: http://mywiki.wooledge.org/Bashism

### CODE REVIEW

File: scripts/quick_install.sh [hunk 1]
```diff
@@ -1,32 +1,31 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```
Comment (maxice8) [inline scripts/quick_install.sh]: we can use /bin/sh as it is defined as an available path in posix

File: scripts/quick_install.sh [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
 
-if ! _can_install && [[ $EUID != 0 ]]; then
+if ! _can_install && [ "$(id -u)" != 0 ]; then
   echo "Run script as sudo"
   exit 1
 fi
 
 if ! _can_install; then
-  echo "Can't install to $BINDIR"
+  echo "Can't install to ${BINDIR}"
```
Comment (maxice8) [inline scripts/quick_install.sh]: `echo` isn't standard but `printf` is

```sh
    printf -- "Can't install to %s" "$BINDIR"
```

File: scripts/quick_install.sh (hunk 1 above)
Comment (profclems) [reply scripts/quick_install.sh]: Okay sure I will change that

File: scripts/quick_install.sh (hunk 2 above)
Comment (profclems) [reply scripts/quick_install.sh]: Thanks. Fixed that

**Pull request merged**